### PR TITLE
Fix platform_tests/test_advanced_reboot.py test_warm_reboot_mac_jump for mac jump detection

### PR DESCRIPTION
**Pull request by byu343**
### Description of PR

Summary:  Update the regex to detect mac jump because the log level for the message changed
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
The log level for fdbEvent messsage to detect mac jump is changed to NOTICE, so the regex used by the test has to be updated.  

#### How did you do it?
Change the regex for detecting mac jump

#### How did you verify/test it?
Check the test can pass after the change

### CODE REVIEW

**Comment by StormLiangMS** [comment]
hi @vaibhavhd  @saiarcot895 could you help to review?

File: tests/platform_tests/templates/expect_boot_messages [hunk 1]
```diff
@@ -19,7 +19,7 @@ r, ".* NOTICE syncd#syncd.*performWarmRestart: switches defined in warm restart.
 r, ".* NOTICE syncd#syncd.*performWarmRestartSingleSwitch: Warm boot: create switch VID.*"
 r, ".* NOTICE bgp#fpmsyncd.*main: Warm-Restart timer started.*.*"
 r, ".* NOTICE bgp#fpmsyncd.*main: Warm-Restart reconciliation processed..*"
-r, ".* INFO syncd#syncd.*SAI_API_FDB:_brcm_sai_fdb_event_cb.*fdbEvent: (delete \(0\)|0) for mac.*"
+r, ".* NOTICE syncd#syncd.*SAI_API_FDB:_brcm_sai_fdb_event_cb.*fdbEvent: (delete \(0\)|0) for mac.*"
```
Comment (saiarcot895) [inline tests/platform_tests/templates/expect_boot_messages:22]: Can you modify this to either not match the log level (as done in the other file) or allow both INFO and NOTICE?

**Comment by saiarcot895** [comment]
Can this PR first go into master branch, and then cherry-picked to 202405 and 202411?

**Comment by vaibhavhd** [comment]
> Can this PR first go into master branch, and then cherry-picked to 202405 and 202411?

I agree. @byu343 , can you please work to solve the conflicts on master and create a PR on master first. We should not create feature branch only PRs, particularly when master is impacted too.

File: tests/platform_tests/templates/expect_boot_messages (hunk 1 above)
Comment (byu343) [reply tests/platform_tests/templates/expect_boot_messages]: Done by removing log level

**Comment by byu343** [comment]
> > Can this PR first go into master branch, and then cherry-picked to 202405 and 202411?
> 
> I agree. @byu343 , can you please work to solve the conflicts on master and create a PR on master first. We should not create feature branch only PRs, particularly when master is impacted too.

This PR has been rebased to master. My earlier conflict issue probably is related to some outdated state of my workspace.

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Review by saiarcot895 — APPROVED** [review]

**Pull request merged**